### PR TITLE
sql: Fix a couple of typos.

### PR DESCRIPTION
**Pull request by yuzefovich**
Release note: None

### CODE REVIEW

File: pkg/sql/window.go [hunk 1]
```diff
@@ -430,11 +429,11 @@ func constructWindowDef(
 //    (for clarity, we ignore grouping rules)
 //
 //   After extractWindowFunctions is run:
-//    source plan's renders: [a, max(b), d, f]
-//    window plan's renders: [nil, nil, c + max(d) + first_value(@3) OVER (PARTITION BY @4)]
+//    source plan's renders: [a, max(b), e]
+//    window plan's renders: [nil, nil, c + max(d) + first_value(@3) OVER (PARTITION BY f)]
 //
```
Comment (nvanbenschoten) [inline pkg/sql/window.go]: Mind adding in the `constructWindowDefinitions` step we just found out about?

**Comment by yuzefovich** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---

*[pkg/sql/window.go, line 434 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/26407#-LEBqRPd8MNVeuUvcOs6-r1-434:-LEBtRMQ3TQ5gaWf63fm:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/03350938059a114096a2894a54512788663b0c63/pkg/sql/window.go#L434)):*
<details><summary><i>Previously, nvanbenschoten (Nathan VanBenschoten) wrote…</i></summary><blockquote>

Mind adding in the `constructWindowDefinitions` step we just found out about?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/26407)*

**Comment by nvanbenschoten** [comment]
:lgtm: thanks!

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/26407#-:-LEBvX1n1Dr8eDEL_9Yz:brfqlux)*

**Pull request merged**